File: 백준/Bronze/2920. 음계/음계.c
#include <stdio.h>

int main(void) {  
   int arr[8];
   for(int i = 0; i < 8; i++){
      scanf("%d", &arr[i]);
   }

   int isAecending = 1, isDescending = 1;
   for(int i = 0; i < 8; i++){
      if (arr[i] != i + 1){
         isAecending = 0;
      }
      if (arr[i] != 8 - i){
         isDescending = 0;
      }
   }

   if(isAecending){
      printf("ascending");
   }
   else if(isDescending){
      printf("descending");
   }
   else{
      printf("mixed");
   }
   
   return 0;
}

File: 백준/Bronze/2920. 음계/음계.py
arr = list(input().split())

flag_a = True
flag_b = True
for i in range(len(arr)):
  if(int(arr[i]) != i + 1):
    flag_a = False
  if(int(arr[i]) != 8 - i):
    flag_b = False

if(flag_a):
  print("ascending")
elif(flag_b):
  print("descending")
else:
  print("mixed")